service/history/queue/transfer_queue_processor_base.go

```diff
@@ -196,7 +211,11 @@ processorPumpLoop:
 		case <-t.shutdownCh:
 			break processorPumpLoop
 		case <-t.notifyCh:
-			t.upsertPollTime(defaultProcessingQueueLevel, time.Time{})
+			t.queueCollectionsLock.RLock()
+			for _, queueCollection := range t.processingQueueCollections {
```

Comment:
Maybe mention why do you need to insert poll time to each queue?

Reply:
Added comments in both tx and timer processor base